setup.py

```diff
@@ -1,38 +1,32 @@
-import os
 import re
-
+from pathlib import Path
 from setuptools import find_packages, setup
 
-# NOTE: If updating requirements make sure to also check Pipfile for any locks
-# NOTE: When updating botocore make sure to update awscli/boto3 versions below
-install_requires = [
-    # pegged to also match items in `extras_require`
-    'botocore>=1.33.2,<1.33.14',
-    'aiohttp>=3.7.4.post0,<4.0.0',
-    'wrapt>=1.10.10, <2.0.0',
-    'aioitertools>=0.5.1,<1.0.0',
-]
+_this_dir = Path(__file__).parent
 
+# NOTE: When updating botocore make sure to update install-requires.txt
```

Comment:
We would need to also update documentation elsewhere.

Relying on simultaneous updates to multiple files is also more error prone IMO.

Reply:
reverted file